Client/Ecosia/UI/APNConsent/UnleashAPNConsentViewModel.swift

```diff
@@ -0,0 +1,75 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Core
+
+// MARK: - UnleashAPNConsentViewModel
+
+/// Final class implementing the APNConsentViewModelProtocol for a specific variant.
+final class UnleashAPNConsentViewModel: APNConsentViewModelProtocol {
+    
+    // MARK: Properties
+    
+    /// Title for the APN consent view, currently based on the Unleash variant.
+    var title: String {
+        titleMatchingVariant
```

Comment:
What's the thought behind creating all those private variables instead of adding the code directly to the protocol implementation? I understand this might be to make it more readable but somehow for me it achieves the opposite with having to scroll up and down 🙈

Reply:
I made it purely for readability purposes. If you think that wouldn't add much value, no strong opinions on that and happy to replace it with concrete implementation

Reply:
It might be a bit personal, I just don't think it's too much code or too complex that it would need to be split for readability and would vote in favour of keeping it together. Not a blocker though 👍